game_eggs/beamng/beammp/beammp.json

```diff
@@ -4,56 +4,92 @@
         "version": "PTDL_v1",
         "update_url": null
     },
-    "exported_at": "2021-12-22T09:10:34+00:00",
-    "name": "BeamMP Servers",
-    "author": "noah@noahserver.online",
-    "description": "This is the server for the multiplayer mod BeamMP for the game BeamNG.drive. The server is the point throug which all clients communicate. You can write lua mods for the server, detailed instructions on the BeamMP Wiki.",
+    "exported_at": "2022-04-17T16:09:38-05:00",
+    "name": "BeamMP",
+    "author": "me@grnt.dev",
```

Comment:
Please do not update these values. They are meant to point at the original dev.

Reply:
Got it, thank you. Changed back to the original values.